### PR TITLE
CLDR-16712 GenerateDtd and CLDRModify passes before v44 general submission

### PR DESCRIPTION
**Pull request by pedberg-icu**
CLDR-16712

- [x] This PR completes the ticket.

**Note: This should be merged with a merge commit (temporarily enable in settings), not the normal squash and merge.**

GenerateDtd and CLDRModify passes before v44 general submission:
- GenerateDtd just moved one element (foreignSpaceReplacement)
- CLDRmodify just moved the new emoji 15.1 annotations into correct order (and updated copyrights, removed blank header lines)
- CLDRModify -fQ just added the tts name to the regular annotations for the new 15.1 emoji
- CLDRModify -fP normalized, sorted, and re-spaced the UnicodeSets for exemplars and parseLenients. This affected several hundred files.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: common/annotations/en.xml [hunk 1]
```diff
@@ -736,6 +736,11 @@ annotations.
 		<annotation cp="😂" type="tts">face with tears of joy</annotation>
 		<annotation cp="🙂">face | slightly smiling face | smile</annotation>
 		<annotation cp="🙂" type="tts">slightly smiling face</annotation>
+		<annotation cp="🙂‍↕">head shaking vertically | nod, yes</annotation> <!-- 1F642 200D 2195 -->
+		<annotation cp="🙂‍↕" type="tts">head shaking vertically</annotation>
+		<!-- Generated lines from Emoji Data v15.1, using GenerateCldrData.java -->
```
Comment (btangmu) [inline common/annotations/en.xml:741]: This comment "Generated lines from Emoji Data v15.1, using GenerateCldrData.java" is no longer associated with a specific set of lines due to the re-ordering; doubtful whether it makes sense to keep the comment in the data; not a big deal

File: common/main/af.xml [hunk 2]
```diff
@@ -1024,11 +1024,11 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		</codePatterns>
 	</localeDisplayNames>
 	<characters>
-		<exemplarCharacters>[a á â b c d e é è ê ë f g h i î ï j k l m n o ô ö p q r s t u û v w x y z]</exemplarCharacters>
-		<exemplarCharacters type="auxiliary">[à å ä ã æ ç í ì ó ò ú ù ü ý]</exemplarCharacters>
+		<exemplarCharacters>[aáâ b c d eéèêë f g h iîï j k l m n oôö p q r s t uû v w x y z]</exemplarCharacters>
+		<exemplarCharacters type="auxiliary">[àåäã æ ç íì óò úùü ý]</exemplarCharacters>
```
Comment (btangmu) [inline common/main/af.xml:1028]: I don't know the rationale for spaces in and around "b c d" but no spaces in "aáâ"

Does this rule only apply in the xml but not in the ST interface? It doesn't seem to be mentioned at https://cldr.unicode.org/translation/core-data/exemplars

File: common/annotations/en.xml [hunk 3]
```diff
@@ -1847,6 +1860,8 @@ annotations.
 		<annotation cp="🐥" type="tts">front-facing baby chick</annotation>
 		<annotation cp="🐦">bird</annotation>
 		<annotation cp="🐦" type="tts">bird</annotation>
+		<annotation cp="🐦‍🔥">fantasy, firebird, rebirth, reincarnation | phoenix</annotation> <!-- 1F426 200D 1F525 -->
```
[the remark this answers is not in the document]
Comment (pedberg-icu) [reply common/annotations/en.xml:1863]: I think this should probably be vertical bar. But this PR is just about moving that entry from one place to another in the file. @macchiati Do we need a ticket to fix the annotation?

File: common/annotations/en.xml (hunk 1 above)
Comment (pedberg-icu) [reply common/annotations/en.xml:741]: Yes, I will fix that in another PR, thanks

File: common/dtd/ldml.dtd [hunk 4]
```diff
@@ -3201,6 +3191,15 @@ CLDR data files are interpreted according to the LDML specification (http://unic
...
 
+<!ELEMENT foreignSpaceReplacement ( #PCDATA ) >
+<!ATTLIST foreignSpaceReplacement xml:space (default | preserve) "preserve" >
+    <!--@METADATA-->
+<!ATTLIST foreignSpaceReplacement alt NMTOKENS #IMPLIED >
+    <!--@MATCH:literal/variant-->
+<!ATTLIST foreignSpaceReplacement draft (approved | contributed | provisional | unconfirmed) #IMPLIED >
+    <!--@METADATA-->
+<!ATTLIST foreignSpaceReplacement references CDATA #IMPLIED >
+    <!--@METADATA-->
```
[the remark this answers is not in the document]
Comment (pedberg-icu) [reply common/dtd/ldml.dtd:3202]: Agreed. We should discuss in Weds meeting.

File: common/main/af.xml (hunk 2 above)
Comment (pedberg-icu) [reply common/main/af.xml:1028]: @macchiati is this related to the exemplar set display changes?

**Pull request merged**